### PR TITLE
Appshots#402

### PR DESCRIPTION
**Pull request by Divyansh-0**
#410 

Added UXarchive to Other section under DevTools

### CODE REVIEW

File: database/backend/architecture.json [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-        "description": "Discover reference architectures, guidance, and best practices for building or migrating your workloads on Google Cloud.",
-        "url": "https://cloud.google.com/architecture/",
-        "category": "backend",
-        "subcategory": "architecture"
-    }
-]
\ No newline at end of file
+  {
+    "name": "Microsoft Architecture Center",
+    "description": "Best practices and patterns for building applications on Microsoft Azure. Also provides suggestions for architecting solutions on Azure by using established patterns and practices.",
```
Comment (CBID2) [inline database/backend/architecture.json:4]:
```suggestion
    "name": "Microsoft Architecture Center",
    "description": "This guide provides best practices and patterns for building applications on Microsoft Azure. It also provides suggestions for architecting solutions on Azure by using established patterns and practices.",
```

File: database/backend/architecture.json [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+    "url": "https://learn.microsoft.com/en-us/azure/architecture/",
+    "category": "backend",
+    "subcategory": "architecture"
+  },
+  {
+    "name": "AWS Architecture Center",
+    "description": "Get Started Architecting on AWS. It provides reference architecture diagrams, vetted architecture solutions, Well-Architected best practices, patterns, icons, and more.",
```
Comment (CBID2) [inline database/backend/architecture.json:11]:
```suggestion
    "description": "Getting started on architecting in AWS? Then check out this source, it provides reference architecture diagrams, vetted architecture solutions, best practices, patterns, icons, and more.",
```

File: database/backend/architecture.json [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
+    "category": "backend",
+    "subcategory": "architecture"
+  },
+  {
+    "name": "Google Cloud Architecture Center",
+    "description": "Discover reference architectures, guidance, and best practices for building or migrating your workloads on Google Cloud.",
+    "url": "https://cloud.google.com/architecture/",
+    "category": "backend",
+    "subcategory": "architecture"
+  }
```
Comment (CBID2) [inline database/backend/architecture.json:29]: Is this a duplicate @Divyansh-0? If not, I highly recommend adding detail to the description to make them more distinctive.

File: database/frontend/design-inspirations.json [hunk 4]
```diff
@@ -1,164 +1,170 @@
...
+  {
+    "name": "Design Vault",
+    "description": "Design Vault is a website that showcases designs from real products, giving you access to the latest and greatest in UI patterns and trends.",
+    "url": "https://designvault.io/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Appshots",
+    "description": "AppShots is a pure, easy-to-use screenshot production tool for App Store applications. You can create beautiful application screenshots for the App Store application screenshots, as well as for promoting and sharing App.",
```
Comment (CBID2) [inline database/frontend/design-inspirations.json:144]:
```suggestion
    "description": "AppShots is an easy-to-use screenshot production tool for App Store applications. You can create beautiful application screenshots, promote, and share your apps.",
```

File: database/frontend/design-inspirations.json [hunk 5]
```diff
@@ -1,164 +1,170 @@
...
+    "url": "https://uxarchive.com/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Page Flows",
+    "description": "The smarter way to research user interface flows, screen, and email best practices and get unstuck is by learning from proven products.",
+    "url": "https://pageflows.com/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
```
Comment (CBID2) [inline database/frontend/design-inspirations.json:161]:
```suggestion
    "description": "The smarter way to research user interface flows, screen, and email best practices, and get unstuck is by learning from proven products.",
    "url": "https://pageflows.com/",
    "category": "frontend",
    "subcategory": "design-inspiration"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Divyansh-0! 😄 I made some suggestions on how to improve your descriptions. Other than that, great work! 😄 👍🏾

**Comment by Divyansh-0** [comment]
> Hi @Divyansh-0! 😄 I made some suggestions on how to improve your descriptions. Other than that, great work! 😄 👍🏾

Hey Thanks for the changes suggested I ll improve all those in a bit.

**Comment by Divyansh-0** [comment]
@CBID2 Made the suggested changes can you review once?

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

**Review by rupali-codes — COMMENTED** [review]
Thanks

**Pull request merged**